src/renderer/renderElement.ts

```diff
@@ -1501,11 +1130,13 @@ export const renderElementToSvg = (
       break;
     }
     case "freedraw": {
-      generateElementShape(element, generator);
-      generateFreeDrawShape(element);
```

Comment:
Calling `generateFreeDrawShape()` was unnecessary as we are calling `getFreeDrawSvgPath(element)` below, directly